Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/EntrycardTimelineComponent.vue

```diff
@@ -285,4 +285,8 @@ div[class*=" row"] {
     padding: 0px;
     color: $primary;
 }
+
+.entry-card-date {
+    font-size: 0.875rem;
+}
```

Comment:
This can be accomplished using a `<small>` tag instead of a `<span>`, or by applying the "small" class which avoids the need for custom CSS.